### PR TITLE
Namespaces

### PR DESCRIPTION
**Pull request by Eomm**
Fixes https://github.com/fastify/fastify-jwt/issues/111

### CODE REVIEW

**Review by mcollina — APPROVED** [review]
lgtm

**Review by Fdawgs — APPROVED** [review]

File: jwt.js [hunk 1]
```diff
@@ -96,10 +97,32 @@ function fastifyJwt (fastify, options, next) {
...
-  fastify.decorateReply('jwtSign', replySign)
+  }
+
+  let jwtVerifyName = 'jwtVerify'
+  let jwtSignName = 'jwtSign'
+  if (namespace) {
+    if (!fastify.jwt) {
+      fastify.decorate('jwt', Object.create(null))
+      fastify.decorateRequest('user', null)
+      fastify.decorateRequest('jwt', Object.create(null))
```
Comment (climba03003) [inline jwt.js:108]: Where does it use? same as `.decorateReply`

Comment (mcollina) [reply jwt.js:108]: I don't understand the question.

Comment (climba03003) [reply jwt.js:108]: @mcollina `jwt` is decorate on `request` and `reply` object but it never used.

Comment (Eomm) [reply jwt.js]: Good spot, it was a misprint

**Review by climba03003 — APPROVED** [review]
LGTM.  

Just advice can the `jwtVerify${namespace}` and `jwtSign${namespace}` capitalize the first character. It is a bit strange to me it the format is not follow `camelCase` Or the reverse way `${namespace}JwtVerify` and `${namespace}JwtSign`.

**Pull request merged**